apimachinery/src/meta/v1.rs

```diff
@@ -455,29 +456,11 @@ pub trait List {
     fn into_items(self) -> Vec<Self::Item>;
 }
 
-impl<'a, T> IntoIterator for &'a List<Item = T> {
```

Comment:
Again, the 2018 syntax highlighted that I didn't really want the original version of this - we want an intermediate Iterator struct rather than IntoIterator on the parent object directly - which the underlying `items().into_iter()` already gives us.